plugins/modules/system/kernel_blacklist.py

```diff
@@ -45,7 +45,7 @@
 import re
 
 from ansible.module_utils.basic import AnsibleModule
-
+from ansible.module_utils._text import to_text
```

Comment:
```suggestion
from ansible.module_utils.common.text.converters import to_text

```
Please keep the double newline between imports and class definition. Also `_text` should no longer be used.